docs/_edot-collector/components.md

```diff
@@ -69,4 +69,55 @@ The EDOT Collector comes with embedded Collector components from the [OTel Colle
 | [httpsprovider ](https://github.com/open-telemetry/opentelemetry-collector/tree/main/confmap/provider/httpsprovider) | [OTel Core Repo](https://github.com/open-telemetry/opentelemetry-collector) | v1.26.0 |
 | [yamlprovider ](https://github.com/open-telemetry/opentelemetry-collector/tree/main/confmap/provider/yamlprovider) | [OTel Core Repo](https://github.com/open-telemetry/opentelemetry-collector) | v1.26.0 |
 
-<!-- end:edot-collector-components-table -->
\ No newline at end of file
+<!-- end:edot-collector-components-table -->
+
+## Core and extended components
+
+The components included in the EDOT Collector are categorized into **[Core]** and **[Extended]** components.
+
+The following table shows the status of each component.
+
+
+| **Component**                | **GitHub Repo**        | **EDOT Col 9.x**           |
```

Comment:
should we maybe just merge that last column into the table above (and disable for now the generation of the above table) until we included the `core` vs `extended` information in the table automation?

Otherwise those two tables are identical (apart from the core vs. extended column)

Reply:
Agreed!